Fix part of #15968 Replace Material Icons with Font Awesome icons

PVenkatArvind opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes or fixes part of #[15968].
2. This PR does the following: [Replace Material Icons with Font Awesome icons]

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The linter/Karma presubmit checks have passed locally on your machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [x] The PR is made from a branch that's **not** called "develop".

## Proof that changes are correct
oppia/core/templates/pages/admin-page/navbar/admin-navbar.component.html 

1)
before 

<img width="355" alt="before" src="https://user-images.githubusercontent.com/79320301/213496566-54062ad6-5068-4f5d-aac7-ce9c6e22b100.png">

after

<img width="322" alt="after" src="https://user-images.githubusercontent.com/79320301/213495538-e7ac3e04-66f3-4e28-8b18-1f6dac5bc62d.png">

2)
 oppia/core/templates/pages/contributor-dashboard-admin-page/navbar/contributor-dashboard-admin-navbar.component.html

after
<img width="119" alt="after_contributor" src="https://user-images.githubusercontent.com/79320301/213495861-6d820888-8439-4c68-8218-c5142309d486.png">


3) oppia/core/templates/pages/exploration-editor-page/translation-tab/state-translation-editor/state-translation-editor.component.htm

before

<img width="126" alt="before3 1" src="https://user-images.githubusercontent.com/79320301/213496647-28abfd4d-8f89-4903-853b-3cc5a0679f3c.png">

after
<img width="42" alt="after_state1" src="https://user-images.githubusercontent.com/79320301/213496780-c7ea44b2-8342-4117-8f4b-730319d57e34.png">


<img width="59" alt="after_State" src="https://user-images.githubusercontent.com/79320301/213496863-cfd58315-7b97-4f5b-8456-f69b8dacb910.png">

<!--
Add videos/screenshots of the user-facing interface to demonstrate that the changes
made in this PR work correctly. If this PR is for a developer-facing feature,
provide the videos/screenshots of developer-facing interface.

Please also include videos/screenshots of the developer tools
browser console, so that we can be sure that there are no console errors.

If the changes in your PRs are autogenerated via a script and you cannot provide proof
for the changes then please leave a comment "No proof of changes needed because {{Reason}}"
and remove all the sections below.
-->

#### Proof of changes on desktop with slow/throttled network

<!--
Make sure to properly verify that everything works correctly, and that there are
no weird UI mistakes or other problems. Also, if there are any newly added fields,
try to fill them out and test that different inputs are correctly accepted/rejected.

Throttle the network (to 3G) using the browser Developer Tools (see references below).
There should be no performance or UI issues while the network is slow.

References:
 - Chrome: https://css-tricks.com/throttling-the-network/
 - Firefox: https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor/Throttling
-->

#### Proof of changes on mobile phone

<!--
In some cases this is not needed (e.g. for pages that we do not expect to
support mobile phones, or for backend-only features).


Feel free to use the Developer Tools emulator for this.

References:
 - Chrome: https://developer.chrome.com/docs/devtools/device-mode/
 - Firefox: https://firefox-source-docs.mozilla.org/devtools-user/index.html#responsive-design-mode
-->

#### Proof of changes in Arabic language

<!--
If the PR changes the UI, make sure to add screenshots with the site
language set to Arabic as well (we use Arabic as it is a language written from right to left).
-->

## PR Pointers

- Make sure to follow the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Make-a-pull-request).
- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL". Oppiabot will help assign that person for you.
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia's-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays.
- Never force push. If you do, your PR will be closed.
- Some of the e2e tests are flaky, and can fail for reasons unrelated to your PR. We are working on fixing this, but in the meantime, if you need to restart the tests, please check the ["If your build fails" wiki page](https://github.com/oppia/oppia/wiki/If-CI-checks-fail-on-your-PR).

ayushrathore111 commented:
Hello @PVenkatArvind I want to talk you regarding this organization codebase. I'm noob in open source so please connect to me :
https://www.linkedin.com/in/ayush-rathore-5404b0223/ here.

Lawful2002 commented on core/templates/pages/admin-page/navbar/admin-navbar.component.html:
```diff
@@ -4,7 +4,6 @@
     background-color: #00376d;
     color: #fff;
   }
-
```
Why is change required?

Lawful2002 replied:
I don't think this has been answered.

PVenkatArvind replied:
Greetings! 
@Lawful2002 the reason I removed that was due to lint checks , As you know that lint checks would fail on space or trailing spaces and also last time a made mistake and you guided me regarding the lexicographical ordered letters to , due to lint checks I have removed because trailing space resulted in lint check failure

Lawful2002 commented on core/templates/pages/admin-page/navbar/admin-navbar.component.html:
```diff
@@ -170,7 +169,8 @@
                   </span>
                   <div *ngIf="!profilePictureDataUrl" class="dropdown-toggle">
-                    <i class="material-icons md-40 oppia-dropdown-toggle-icon">&#xE853;</i>
+                    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
I don't think you need to import the stylesheet from CDN, you can directly use the Font Awesome classes. Remove this import from here and everywhere else.

PVenkatArvind replied:
Okay!! @Lawful2002 Thanks!

Lawful2002 commented on core/templates/pages/contributor-dashboard-admin-page/navbar/contributor-dashboard-admin-navbar.component.html:
```diff
@@ -74,6 +74,7 @@
         </ul>
       </div>
       <div class="navbar-header float-right">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Ditto.

PVenkatArvind replied:
Okay!! @Lawful2002 Thanks!

Lawful2002 commented on core/templates/pages/exploration-editor-page/translation-tab/state-translation-editor/state-translation-editor.component.html:
```diff
@@ -1,17 +1,18 @@
 <div class="oppia-translated-content-header">Translated content</div>
 <div class="oppia-state-translation-editor-section oppia-editor-card-section">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Ditto.

PVenkatArvind replied:
Okay!! @Lawful2002 Thanks!

Lawful2002 reviewed (changes requested):
Thanks @PVenkatArvind, left some comments, PTAL.

Lawful2002 reviewed (commented):
@PVenkatArvind, a note: please don't resolve comments on your own as this causes confusion when taking another pass. Also one question I initially had wasn't answered, PTAL.

PVenkatArvind commented:
> @PVenkatArvind, a note: please don't resolve comments on your own as this causes confusion when taking another pass. Also one question I initially had wasn't answered, PTAL.

Greetings! @Lawful2002 Noted! , Sorry for the inconvenience caused .
@Lawful2002 the reason I removed that was due to lint checks , As you know that lint checks would fail on space or trailing spaces and also last time a made mistake and you guided me regarding the lexicographical ordered letters keeping all this in mind I made a change, due to lint checks I have removed because trailing space resulted in lint check failure

PVenkatArvind commented:
Greetings! @Lawful2002 I have addressed the changes mentioned please review

chris7716 commented:
@PVenkatArvind could you please put a before and after of contributor dashboard admin navbar changes? Cuz this is what I currently see
![image](https://user-images.githubusercontent.com/41409951/214496428-eb37c0d1-bcee-49f8-a67b-62b0a5284bf7.png)

chris7716 reviewed (changes requested):
@PVenkatArvind could you please put a before and after of contributor dashboard admin navbar changes? Cuz this is what I currently see
![image](https://user-images.githubusercontent.com/41409951/214496428-eb37c0d1-bcee-49f8-a67b-62b0a5284bf7.png)

PVenkatArvind commented:
> @PVenkatArvind could you please put a before and after of contributor dashboard admin navbar changes? Cuz this is what I currently see ![image](https://user-images.githubusercontent.com/41409951/214496428-eb37c0d1-bcee-49f8-a67b-62b0a5284bf7.png)


Greetings!!
@chris7716  @Lawful2002 

before

<img width="212" alt="before_contrib" src="https://user-images.githubusercontent.com/79320301/214567263-d93acd76-b0f4-4237-873c-86173a295678.png">

after

<img width="175" alt="after_contrib" src="https://user-images.githubusercontent.com/79320301/214567370-6a5418fe-d1ee-436e-be7c-966450f1ebfd.png">

Lawful2002 reviewed (approved):
LGMT! Thanks

PVenkatArvind commented:
Greetings!! Resolved Merge conflicts!!

PVenkatArvind commented:
<img width="961" alt="merge_conflict2" src="https://user-images.githubusercontent.com/79320301/215482801-1cac378b-0e6c-4f48-82e9-3a9973a76038.png">


<img width="715" alt="after_merge" src="https://user-images.githubusercontent.com/79320301/215482853-5e8a68b1-fd50-45c8-b51b-65ac915ec811.png">

chris7716 commented on core/templates/pages/contributor-dashboard-admin-page/navbar/contributor-dashboard-admin-navbar.component.html:
```diff
@@ -93,7 +93,7 @@
                   </span>
                   <div *ngIf="!profilePictureDataUrl" class="dropdown-toggle">
-                    <i class="material-icons md-40 oppia-dropdown-toggle-icon">&#xE853;</i>
+                    <i class=" fas fa-user-edit md-40 oppia-dropdown-toggle-icon"></i>
```
```suggestion
                    <i class="fas fa-user-edit md-40 oppia-dropdown-toggle-icon"></i>
```

chris7716 reviewed (changes requested):
@PVenkatArvind Just a small nit in the code.

But your screenshots look a bit weird for me. Could you please provide the whole screenshot of contributor dashboard admin page to see how it looks?
